### ISSUE TITLE
In order to [empower customers], I want [feedback reported by couriers to be accessible to customers].

### ISSUE THREAD

**Issue by rayon9coop** [coopcycle/coopcycle-web#3466]
**In order to**  `achieve something`
Sometimes we need to report information to customers (= owner of a ...coopcycle.org/admin/stores) : absent recipient, delivery to another location,... or information related to the delivery: wrong number of parcels, higher weight than expected, recepient will move next month.... 

Couriers can already note this information in the comments field (in the courier application) but shop owners cannot see these comments. 
So if they need to understand e.g. the reasons for delivery incidents, they have to ask dispatch. If they didn't have to ask, it could save everyone's time. 

**As a** dispatch
I don't want to take time to answer on the phone or to emails to communicate some info about delivery if these info can be accessible to shop's owner in the app

**I want** `something to be added or changed`
It will be helpfull to add a reporting page to the STORE with failed/successful deliveries and some info:
- comments
- link to pictures/signatures
- tarification info
- ...
in short, all the info relevant for store's owner.

**Comment by ERAMANrepartos** [comment]
Wooof this is a big one!
Really what we are talking about here is the implementation of the "incidents" feature, like all last mile logistics platforms have. This is a feature we have to create for other reasons, is the good news. That said, I am going to close this issue to how specific it is, but in the coming days will open a new issue about "incidents" and how they must work that will cover these use cases
